docs/docs/development/contributing.md

```diff
@@ -0,0 +1,240 @@
+# Development Guide
+
+## Setting Up Development Environment
+
+### Prerequisites
+
+1. Install build dependencies:
+```bash
+# Linux
+sudo apt-get install python3-dev cmake build-essential
+
+# macOS
+brew install cmake python
+
+# Windows
+# Install Visual Studio Build Tools and CMake
+```
+
+2. Clone the repository:
+```bash
+git clone https://github.com/tinix84/PyMKF.git
+cd PyMKF
+```
+
+3. Create virtual environment:
+```bash
+python -m venv venv
+source venv/bin/activate  # Linux/macOS
+.\venv\Scripts\activate   # Windows
+```
+
+4. Install development dependencies:
+```bash
+pip install -e ".[dev]"
+```
+
+## Project Structure
+
+```
+PyMKF/
+├── src/pymkf/           # Source code
```

Comment:
this is not the current strcture, right?

Reply:
yes sorry is a typo i will fix it